include/maths/CMathsFuncs.h

```diff
@@ -54,7 +54,7 @@ class MATHS_EXPORT CMathsFuncs : private core::CNonInstantiatable {
     //! Check if an element is NaN.
     static bool isNan(const core::CSmallVectorBase<double>& val);
 
-    //! Wrapper around boost::math::isinf() which avoids the need to add
+    //! Wrapper around std::isinf() which avoids the need to add
     //! cryptic brackets everywhere to deal with macros.
```

Comment:
Based on your other changes in this PR we no longer need to add cryptic brackets everywhere to deal with macros, so this comment can be removed too.